hibernate-core/src/main/java/org/hibernate/dialect/CUBRIDDialect.java

```diff
@@ -36,7 +36,6 @@
 	 * Constructs a CUBRIDDialect
 	 */
 	public CUBRIDDialect() {
-		super();
 
```

Comment:
The above statement is unnecessary for JVM has done that for us automatically

Reply:
It is done automatically, yes.  All of us have developed Java for more than a day ;)

But:
1) imo this is exactly the kind of change that cause unnecessary conflicts (right?)
2) often these are nice for debug breakpoints
3) there is no reason to not have them.  there is no performance penalty

Reply:
Thanks for the explanation. At least point 2 makes sense. I reverted back the unnecessary changes.